.pre-commit-config.yaml

```diff
@@ -33,3 +33,5 @@ repos:
           - '--args=--only=terraform_required_providers'
           - '--args=--only=terraform_standard_module_structure'
           - '--args=--only=terraform_workspace_remote'
+      - id: terraform_tfsec
+        files: ^examples/ # only scan `examples/*` which are the implementation
```

Comment:
👍 

Some tools evaluate the code of the modules instead of the implementations and blame us for bad code... lol